art/metrics/metrics.py

```diff
@@ -73,12 +78,53 @@ def get_crafter(classifier: "CLASSIFIER_TYPE", attack: str, params: Optional[Dic
     except Exception:  # pragma: no cover
         raise NotImplementedError(f"{attack} crafting method not supported.") from Exception
 
+    if "params" in SUPPORTED_METHODS[attack]:
+        crafter.set_params(**SUPPORTED_METHODS[attack]["params"])
     if params:
         crafter.set_params(**params)
 
     return crafter
 
 
+def adversarial_accuracy(
+    classifier: "CLASSIFIER_TYPE",
+    x: np.ndarray,
```

Comment:
Should we also provide the option to provide true labels `y`?

Reply:
Added the option.